### PR TITLE
Fix the invalid date bug

### PR DESCRIPTION
**Pull request by Maheima**
### Issue:
Currently, on the cards where startedOn isn't available, it shows invalid date. We need to change it to N/A instead
Fixes: #899 

### Description:
Wrote a function that gets from startedon date only when we get started on date of that task from the API

### Files changed:
__tests__/Unit/Components/Tasks/TaskDetails.test.tsx -> Added tests cases
src/components/taskDetails/index.tsx ->  Fixed the NA to N/A

### Dev Tested:
- [x] Yes

Video of the change:
https://www.loom.com/share/031d00dd92c34ffa8cb8cb0bb53fa978?sid=ec61421a-c7ee-44b8-b7f3-cdc1665b12cd

### Test coverage:

Before:

image

After:

image

### CODE REVIEW

File: src/components/taskDetails/index.tsx [hunk 1]
```diff
@@ -158,8 +158,12 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
         }
     }
 
-    function getTaskEstimates(timestamp: number | undefined) {
-        return timestamp ? convertTimeStamp(timestamp) : 'NA';
+    function getStartedOn(timestamp: number | undefined) {
+        return timestamp ? convertTimeStamp(timestamp) : 'N/A';
```
Comment (iamitprakash) [inline src/components/taskDetails/index.tsx:162]: what if timestamp undefined ...?

File: src/components/taskDetails/index.tsx [hunk 2]
```diff
@@ -158,8 +158,12 @@ const TaskDetails: FC<Props> = ({ taskID }) => {
...
+    }
+
+    function getEndsOn(timestamp: number | undefined) {
+        return timestamp ? convertTimeStamp(timestamp) : 'TBD';
```
Comment (iamitprakash) [inline src/components/taskDetails/index.tsx:166]: what if timestamp is undefined ...?

**Review by iamitprakash — CHANGES_REQUESTED** [review]

File: src/components/taskDetails/index.tsx (hunk 1 above)
Comment (Maheima) [reply src/components/taskDetails/index.tsx:162]: But that is what I'm checking for . If it is undefined, it goes to the else condition i.e., N/A

**Review by iamitprakash — APPROVED** [review]

**Review by Achintya-Chatterjee — APPROVED** [review]
LGTM

**Pull request merged**